packages/shared/lib/participation/stores.ts

```diff
@@ -143,19 +144,20 @@ export const partiallyUnstakedAmount: Readable<number> = derived(
     ([$participationOverview, $partiallyStakedAccounts]) => {
         if ($partiallyStakedAccounts.length <= 0) return 0
 
-        const partiallyStakedOverviews = $partiallyStakedAccounts.map((psa) =>
-            $participationOverview.find((apo) => apo.accountIndex === psa.index)
-        )
-        const assemblyUnstakedFunds = partiallyStakedOverviews.reduce(
-            (total, apo) => total + apo.assemblyUnstakedFunds,
-            0
-        )
-        const shimmerUnstakedFunds = partiallyStakedOverviews.reduce(
-            (total, apo) => total + apo.shimmerUnstakedFunds,
-            0
-        )
+        const _eval = (overview: AccountParticipationOverview): number => {
```

Comment:
The formatting is a bit bizarre for this function, but it is still "formatted" according to Prettier.